pkgs/development/python-modules/pytest-pytestrail/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools-scm
+, pytest
+, testrail-api
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-pytestrail";
+  version = "0.10.5";
+
+  SETUPTOOLS_SCM_PRETEND_VERSION = version;
+
+  src = fetchFromGitHub {
+    owner = "tolstislon";
+    repo = "pytest-pytestrail";
+    rev = version;
+    sha256 = "sha256-y34aRxQ8mu6b6GBRMFVzn1shMVc7TumdjRS3daMEZJM=";
+  };
+
+  nativeBuildInputs = [
+    setuptools-scm
+  ];
+
+  propagatedBuildInputs = [
+    pytest
```

Comment:
We never propagate pytest because it makes overwriting it hard. Please move it to buildInputs

Reply:
Thank you for the feedback.